### ISSUE TITLE
Underwater Radios

### ISSUE THREAD

**Issue by Soldia1138** [IDI-Systems/acre2#14]
Add underwater radios (and underlying calculations)

A few points to be considered:
- Underwater Communication is not trivial
- Only low frequency communication is useful
- Investigation on Radios and
- their realization are part of this

**Comment by TheMagnetar** [comment]
I can also look at new technologies performed in Tsunami detection in Japan. The devices communicate using acoustic waves, which later are transmitted to radio waves by a buoy on the surface. I have to check how feasible could that be or if there is interest in that.

**Comment by Sniperhid** [comment]
Tbh I think if/when we do this it would be best to start with a 3D model of a diver radio and get the interface rigged up as a usable radio and then implement the back-end for underwater radios. As they are pretty niché. But if people want to volunteer, don't let me get your in way!

**Comment by jonpas** [comment]
3D model is step 1, if you or someone can/wants to make one, go for it! Reach us on our Slack so we can get the model and details down. ;)

After that is backend, underwater radio is quite different, and requires different simulation aspects.

**Comment by markooff** [comment]
Yes, I can understand this "little difference" in propagation of radio signals and frequencies in two such different environments like open air and underwater ...  OK but which type of radio/underwater intercom should it be (for making a model) ? I know a bit about some military radios and their manufactures (like Harris, Rhode &Schwartz and so on ) but almost none about underwater. If you can provide me with some advice , ideas ?  
(ASAF for the communication with for instance the submarine deep underwater they use an ELF (Extremely Low Frequency) radio antenas and sets . ) I've also heard about the ULF and VLF radio technology for the underwater voice communication . But that's all .

**Comment by jonpas** [comment]
@IDI-Systems/acre-developers someone else can probably answer that for you, I don't know myself. I suggest you hop on our Slack, can get information a lot faster there. :)

**Comment by StickHogue** [comment]
Bear in mind that *hardwired* underwater comms is a thing, boat-to-diver. (e.g. [this](http://www.underwatersystems.net/underwater-comms.html)) That alone would be very cool to have, with or without "ropes" to represent the cables. (I doubt Arma handles ropes properly underwater anyway.)  It's essentially like a vehicle intercom but at a distance.  Perhaps users could define just how much cable is available on the boat's spool, up to some reasonable limit like 200m, and it would be good to have some indication of how much cable remains available as you swim away.  (I'm assuming people who use such systems can attach buoyancy floats to the cables so divers aren't weighed down by long cables, but I don't know that - maybe they only do wired underwater comms directly underneat boats.  But I'd *want* to allow long cables for what I have in mind...)  Thanks!

**Comment by mrschick** [comment]
@StickHogue the wired comms sound like a good idea, would be easy enough to model and visualize with the WIP feature of #1290. I quickly tested it by adding an infantry phone to the SDV's config, looks like this and works just like on land (infPhone origin point needs to be adapted of course).

![sdv-infantry-phone-test](https://github.com/IDI-Systems/acre2/assets/58027418/0c7a3d0d-d8cf-482b-afaa-8eb4dcce35d9)
The only things missing would be support for multiple divers being able to use the infantry phone at the same time, as well as not breaking the rope when going out of range (could be done by making the spawned rope unbreakable and the PFH not disconnect when out of range).
I also can't think of many underwater operations in Arma where you would realistically use wired comms over wireless ones, except for mine-clearing operations supported by a boat, where (just like above water) wireless comms probably risk detonating the device.

**As for the wireless-comms aspect:**
Basic research suggests that radio propagation for diver-to-diver comms is so poor that it makes more sense to use "radios" that use sound waves.
I only have a modest understanding of radio wave propagation (especially underwater or crossing the surface) IRL and how ACRE simulates it, but would break this feature down into the following points.

- Modelling the new underwater radio based on one available for the civilian market, like [this](https://www.oceantechnologysystems.com/learning-center/through-water-communications/).
- Modelling of wireless underwater-to-underwater sound-wave comms using the newest [NATO JANUS Standard](https://www.nato.int/cps/bu/natohq/news_143247.htm), which will be used by civilian entities as well and probably has the most propagation documentation available.
- Modelling of surface-to-underwater comms using normal radios could be done based on underwater receiver depth, at least for Arcade mode a few meters sounds like it could be a good approximation, though depending on how radio waves are reflected by the water's surface that could be too permissible. Transmission from underwater (using a normal radio) to the surface could be blocked entirely since that radio's microphone would not be integrated in the rebreather's mouthpiece, while allowing transmission when surfaced.
- SDVs could have Racks for conventional and underwater comms, while only allowing conventional comms if surfaced or with its sensor mast breaching the surface (could be checked by depth and [`periscopeElevation`](https://community.bistudio.com/wiki/periscopeElevation)).
